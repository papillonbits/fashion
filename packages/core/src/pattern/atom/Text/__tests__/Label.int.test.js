import {
  blackTiny,
  blackSmall,
  blackMedium,
  blackLarge,
  blueTiny,
  blueSmall,
  blueMedium,
  blueLarge,
  greyTiny,
  greySmall,
  greyMedium,
  greyLarge,
} from './Label.int.story'

describe('<Label />', () => {
  describe('Render', () => {
    test('must match blackTiny', () => {
      expect(global.renderToJSON(blackTiny())).toMatchSnapshot()
    })

    test('must match blackSmall', () => {
      expect(global.renderToJSON(blackSmall())).toMatchSnapshot()
    })

    test('must match blackMedium', () => {
      expect(global.renderToJSON(blackMedium())).toMatchSnapshot()
    })

    test('must match blackLarge', () => {
      expect(global.renderToJSON(blackLarge())).toMatchSnapshot()
    })

    test('must match blueTiny', () => {
      expect(global.renderToJSON(blueTiny())).toMatchSnapshot()
    })

    test('must match blueSmall', () => {
      expect(global.renderToJSON(blueSmall())).toMatchSnapshot()
    })

    test('must match blueMedium', () => {
      expect(global.renderToJSON(blueMedium())).toMatchSnapshot()
    })

    test('must match blueLarge', () => {
      expect(global.renderToJSON(blueLarge())).toMatchSnapshot()
    })

    test('must match greyTiny', () => {
      expect(global.renderToJSON(greyTiny())).toMatchSnapshot()
    })

    test('must match greySmall', () => {
      expect(global.renderToJSON(greySmall())).toMatchSnapshot()
    })

    test('must match greyMedium', () => {
      expect(global.renderToJSON(greyMedium())).toMatchSnapshot()
    })

    test('must match greyLarge', () => {
      expect(global.renderToJSON(greyLarge())).toMatchSnapshot()
    })
  })
})
