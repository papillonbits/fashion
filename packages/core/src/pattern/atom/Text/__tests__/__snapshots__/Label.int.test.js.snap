// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Label /> Render must match blackLarge 1`] = `
<div>
  <span
    class="label labelBlack labelLarge"
  >
    black large label
  </span>
</div>
`;

exports[`<Label /> Render must match blackMedium 1`] = `
<div>
  <span
    class="label labelBlack labelMedium"
  >
    black medium label
  </span>
</div>
`;

exports[`<Label /> Render must match blackSmall 1`] = `
<div>
  <span
    class="label labelBlack labelSmall"
  >
    black small label
  </span>
</div>
`;

exports[`<Label /> Render must match blackTiny 1`] = `
<div>
  <span
    class="label labelBlack labelTiny"
  >
    black tiny label
  </span>
</div>
`;

exports[`<Label /> Render must match blueLarge 1`] = `
<div>
  <span
    class="label labelBlue labelLarge"
  >
    blue large label
  </span>
</div>
`;

exports[`<Label /> Render must match blueMedium 1`] = `
<div>
  <span
    class="label labelBlue labelMedium"
  >
    blue medium label
  </span>
</div>
`;

exports[`<Label /> Render must match blueSmall 1`] = `
<div>
  <span
    class="label labelBlue labelSmall"
  >
    blue small label
  </span>
</div>
`;

exports[`<Label /> Render must match blueTiny 1`] = `
<div>
  <span
    class="label labelBlue labelTiny"
  >
    blue tiny label
  </span>
</div>
`;

exports[`<Label /> Render must match greyLarge 1`] = `
<div>
  <span
    class="label labelGrey labelLarge"
  >
    grey large label
  </span>
</div>
`;

exports[`<Label /> Render must match greyMedium 1`] = `
<div>
  <span
    class="label labelGrey labelMedium"
  >
    grey medium label
  </span>
</div>
`;

exports[`<Label /> Render must match greySmall 1`] = `
<div>
  <span
    class="label labelGrey labelSmall"
  >
    grey small label
  </span>
</div>
`;

exports[`<Label /> Render must match greyTiny 1`] = `
<div>
  <span
    class="label labelGrey labelTiny"
  >
    grey tiny label
  </span>
</div>
`;
